www/src/components/Menu.jsx

```diff
@@ -150,6 +157,39 @@ const Menu = () => {
               <Link to="/foundations/css-utilities">CSS Utility Classes</Link>
             </li>
           </ul>
+          <h3 className="mb-4">
+            Playground
+            <Badge className="ml-1 align-bottom" variant="info">Beta</Badge>
+          </h3>
+          <ul className="list-unstyled small mb-5">
+            <li className="mr-3 mb-3">
+              A drag-and-drop UI builder for prototyping with Paragon components
+            </li>
+            <li className="mr-3">
+              <OverlayTrigger
+                trigger="focus"
+                overlay={(
+                  <Popover>
+                    <Popover.Title as="h3">
+                      <Icon src={Issue} />
+                      Сoming soon
+                    </Popover.Title>
+                    <Popover.Content>
+                      Playground is under development
+                    </Popover.Content>
+                  </Popover>
+                )}
+              >
+                <Button
+                  className="mr-2 mb-2"
+                  variant="outline-primary"
+                  onClick={handlePlaygroundClick}
```

Comment:
Let's make this button CTA have `size="sm"` to match the screenshot in the ticket.

Reply:
Added